app.py

```diff
@@ -0,0 +1,23 @@
+import os
+import sqlite3
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+app.config["DEBUG"] = True
+SECRET_KEY = "12345"
```

Comment:
🔐 **Security Issue: Hardcoded Secret Key**

The secret key is hardcoded and weak. This poses a security risk in production environments.

```suggestion
SECRET_KEY = os.environ.get('SECRET_KEY', 'dev-key-change-in-production')
app.config['SECRET_KEY'] = SECRET_KEY
```

Use environment variables for sensitive configuration and generate a strong secret key for production.